cmd/minikube/cmd/start.go

```diff
@@ -156,6 +169,36 @@ func runStart(cmd *cobra.Command, args []string) {
 	clusterUp(&config)
 }
 
+// Set docker and oc cluster proxy in place
+func setProxy() {
+	if viper.IsSet("http-proxy") {
```

Comment:
Should we not add some basic validation to the parameter? This could save the user a quite some pain in case of typos.